src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.java

```diff
@@ -364,11 +364,23 @@
    
    // Tools
    public abstract AppCommand showShellDialog();
-   public abstract AppCommand newTerminal();
    public abstract AppCommand macPreferences();
    public abstract AppCommand showOptions();
    public abstract AppCommand modifyKeyboardShortcuts();
-
+   
+   // Terminal
+   public abstract AppCommand newTerminal();
+   public abstract AppCommand renameTerminal();
```

Comment:
That said there's nothing wrong with using `AppCommands` here (just wanted to make you aware that there's an alternative)